compute/bare-metal-cloud/api-cli/bare-metal-with-cli.mdx

```diff
@@ -42,348 +16,349 @@ You can create and manage your Bare Metal servers from the console, via the API
-  b) explicitly target the fr-par-2 zone in the commands to create and manage your server.
 
-  In this tutorial, we’ll target the zone explicitly in our commands.
+1. Type the following command in your terminal to create your Bare Metal Server:
 
-  </Message>
+  `scw baremetal server create name=name-of-your-server type=GP-BM1-L zone=fr-par-2`
 
-2. Type the following command into your terminal to create a Bare Metal server. You can choose the value for `name`, but remember the server `type` must be one from the list we saw in the previous step, and the zone must be `fr-par-2`:
+  An output similar to the following displays: 
 
   ```
-  scw baremetal server create name=bm-tutorial-server type=GP-BM1-M zone=fr-par-2
-  ```
-
-  <Message type="note">
-  
-  Optionally, you can include additional arguments as per the `ARGS` information we just saw, eg `description` to add a description of your Bare Metal server, or `tags.0` to add tags.
-
-  </Message>
-
-  You should see output in the console showing the API request and response, and finally confirming the ID and details of your server:
-
-  ```
-  ID              <ID of the Bare Metal Server>
-  OrganizationID  <Your Organization ID>
-  ProjectID       <Your Project ID>
-  Name            bm-tutorial-server
+  ID              12345678-1234-1234-1234-123456789123
+  OrganizationID  12345678-1234-1234-1234-123456789123
+  ProjectID       12345678-1234-1234-1234-123456789123
+  Name            name-of-your-server
   Description     -
-  UpdatedAt       now
-  CreatedAt       now
+  UpdatedAt       36 seconds ago
+  CreatedAt       36 seconds ago
   Status          delivering
-  OfferID         <Offer ID>
-  Tags.0          -
+  OfferID         12345678-1234-1234-1234-123456789123
   Domain          -
   BootType        normal
   Zone            fr-par-2
   PingStatus      unknown
   ```
 
-3. Note down the ID of your Bare Metal server, as you will need it in the next step.
+  <Message type="note">
+
+    - For more information on creating a Bare Metal server, type `scw baremetal server create --help` in your terminal. 
+    - You can type additional information in your terminal such as `description` to add a description of your Bare Metal server, or `tags.0` to add tags.
+
+  </Message>
+
+2. Write down the ID of your Bare Metal server, as you will need it in the next steps.
 
   <Message type="note">
-  
-  You can view a summary of information for your Bare Metal server, and details for any other Bare Metal servers you have previously created, by typing `scw baremetal server list zone=fr-par-2`. For full information, use the command `scw baremetal server get <Server ID> zone=fr-par-2`
+
+    * Type `scw baremetal server list zone=fr-par-2` for a summary of information for your Bare Metal server, and details for other Bare Metal servers you have previously created. 
```

Comment:
you can sometimes replace Bare Metal server by just server, so the full name repeats less.